README.md

```diff
@@ -138,6 +139,19 @@ schema.cast({
 // => { name: 'jimmy', age: 24, createdOn: Date }
 ```
 
+### Using a custom locale dictionary
+```js
```

Comment:
can you flesh this out a bit more, explanation of the code, why you need to do this, etc.